core/__tests__/tasks/record/makeExports/makeExports.ts

```diff
@@ -0,0 +1,210 @@
+import { helper } from "@grouparoo/spec-helper";
+import { api, config, task, specHelper } from "actionhero";
+import {
+  Group,
+  GrouparooRecord,
+  RecordProperty,
+  Schedule,
+  Source,
+} from "../../../../src";
+
+describe("tasks/record:makeExports", () => {
+  let source: Source;
+  let schedule: Schedule;
+  let group: Group;
+
+  helper.grouparooTestServer({
+    truncate: true,
+    enableTestPlugin: true,
+    disableTestPluginImport: true,
+  });
+  beforeEach(async () => await api.resque.queue.connection.redis.flushdb());
+  beforeAll(async () => await helper.factories.properties());
+
+  beforeAll(async () => {
+    source = await Source.findOne();
+    schedule = await helper.factories.schedule(source);
+  });
```

Comment:
Nitpick: You can combine `beforeAll`. I'm not sure what logic jest uses to order them.